### PR TITLE
Add bedtools getfasta module

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Adding the `bedtools-getfasta` module as discussed [here](https://github.com/nf-core/viralrecon/issues/149)

### CODE REVIEW

File: software/bedtools/getfasta/main.nf [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+def options    = initOptions(params.options)
+
+process BEDTOOLS_GETFASTA {
+    tag "$bed"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
+
+    conda (params.enable_conda ? "bioconda::bedtools=2.30.0" : null)
```
Comment (drpatelh) [inline software/bedtools/getfasta/main.nf:14]:
```suggestion
    conda (params.enable_conda ? "bioconda::bedtools=2.30.0=hc088bd4_0" : null)
```

File: tests/software/bedtools/getfasta/main.nf [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+nextflow.enable.dsl = 2
+
+include { BEDTOOLS_GETFASTA } from '../../../../software/bedtools/getfasta/main.nf' addParams( options: [:] )
+
+workflow test_bedtools_getfasta {
+    def bed,fasta = []
+    bed   = [ file("${launchDir}/tests/data/bed/C.bed", checkIfExists: true) ]
+    fasta =  [ file("${launchDir}/tests/data/fasta/E_coli/NC_010473.fa", checkIfExists: true) ]
+
+    BEDTOOLS_GETFASTA( bed, fasta )
```
Comment (drpatelh) [inline tests/software/bedtools/getfasta/main.nf:12]:
```suggestion
    def bed   = [ file("${launchDir}/tests/data/bed/C.bed", checkIfExists: true) ]
    def fasta =  [ file("${launchDir}/tests/data/fasta/E_coli/NC_010473.fa", checkIfExists: true) ]

    BEDTOOLS_GETFASTA ( bed, fasta )
```

File: tests/software/bedtools/maskfasta/main.nf [hunk 3]
```diff
@@ -5,7 +5,7 @@ nextflow.enable.dsl = 2
 include { BEDTOOLS_MASKFASTA } from '../../../../software/bedtools/maskfasta/main.nf' addParams( options: [:] )
 
 workflow test_bedtools_maskfasta {
-    def input,fasta = []
+    def bed,fasta = []
     bed = [ [ id:'test'],
               file("${launchDir}/tests/data/bed/C.bed", checkIfExists: true) ]
     fasta =  [ file("${launchDir}/tests/data/fasta/E_coli/NC_010473.fa", checkIfExists: true) ]
```
Comment (drpatelh) [inline tests/software/bedtools/maskfasta/main.nf:11]:
```suggestion
    def bed = [ [ id:'test'],
              file("${launchDir}/tests/data/bed/C.bed", checkIfExists: true) ]
    def fasta = [ file("${launchDir}/tests/data/fasta/E_coli/NC_010473.fa", checkIfExists: true) ]
```

**Review by drpatelh — APPROVED** [review]
🎉 Need to add the Conda build id to all files. I forgot to look for those! This is why a linter would be really useful!

**Comment by JoseEspinosa** [comment]
Indeed! I saw the discussion on the tools channel just now. Thanks!

**Pull request merged**